### DISCUSSION TITLE
build.sh fails with error 429

### DISCUSSION THREAD

**Discussion by pminiszewski** [google/orbit#4438] [Q&A]
```
$ ./build.sh 
bzip2/1.0.8: Not found in local cache, looking in remotes...
bzip2/1.0.8: Trying with 'bintray'...
ERROR: Failed requirement 'bzip2/1.0.8' from 'conanfile.py (OrbitProfiler/None@orbitdeps/stable)'
ERROR: 429: status code: 429, reason phrase: You have reached your monthly quota for Data Transfer. To continue and start enjoying unlimited transfer, upgrade your subscription. [Remote: bintray]
```

Guess all the profiling will have to wait until next month.  ¯\_(ツ)_/¯

**Comment by beckerhe** [comment]
Hi pminiszewski,
I'm sorry. We are working on a solution but this will probably take a bit of time.

In the mean time you can install the packages from the repository. The following should work (assuming you're the repo's root directory):

```
./third_party/conan/recipes/export_packages.sh
conan export third_party/conan/recipes/llvm-core 12.0.0@orbitdeps/stable
conan remote disable bintray
./build.sh
```

Note: Make sure not to call `bootstrap-orbit.sh` after disabling the remote with `conan remote disable bintray` as this would re-enable the non-working repository.
